tenzir/tenzir.cpp

```diff
@@ -306,5 +307,16 @@ auto main(int argc, char** argv) -> int {
     render_error(*root, run_error, std::cerr);
     return EXIT_FAILURE;
   }
+  // Wait for up to 60 seconds for all actors to exit, ignore the rest.
+  if (is_server) {
+    for (int i = 0; i < 600; ++i) {
+      if (sys.registry().running() == 0) {
+        break;
+      }
+      std::this_thread::sleep_for(std::chrono::milliseconds{100});
```

Comment:
This sleep will likely cause every `tenzir-node` process to wait for at least 100ms on shutdown. I'm not convinced that this fixed interval is a good idea.

Reply:
`tenzir-node` is not interactive so I don't think this is gonna be a problem.